index.html

```diff
@@ -1,20 +1,450 @@
+  let mediaStream = await navigator.mediaDevices.getDisplayMediaSet({video:true});
+  videoElement0.srcObject = mediaStream[0];
+  videoElement1.srcObject = mediaStream[1];
+  // ...
+} catch (e) {
+  console.log('Unable to acquire screen captures: ' + e);
+}</pre>
+  </section>
+  <section id="terminology">
+    <h2>
+      Terminology
+    </h2>
+    <p>
+      This document uses the definition of {{MediaStream}}, {{MediaStreamTrack}}
+      from [[!GETUSERMEDIA]] and <a data-cite="screen-capture#dfn-display-surface">display surface</a>,
+      <a data-cite="screen-capture#dfn-active-user-consent">active user consent</a>,
+      <a data-cite="screen-capture#dfn-floor-value">floor value</a> from [[!screen-capture]].
+    </p>
+  </section>
+  <section id="cmds">
+    <h2>
+      Capturing multiple display surfaces
+    </h2>
+    <p>
+      Capture of multiple <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>
+      is enabled through the addition of a new
+      {{MediaDevices/getDisplayMediaSet}} method on the {{MediaDevices}}
+      interface, that is similar to {{MediaDevices/getUserMedia()}},
+      except that it acquires media from multiple display devices
+      chosen by the end-user each time.
+    </p>
     <section>
-      <h2>Background</h2>
-      <p>TODO</p>
+      <h2>
+        <dfn>MediaDevices</dfn> Additions
+      </h2>
+      <pre class="idl">
+          partial interface MediaDevices {
+            Promise&lt;sequence&lt;MediaStream&gt;&gt; getDisplayMediaSet(optional DisplayMediaStreamOptions options = {});
+          };
+        </pre>
+      <dl data-link-for="MediaDevices" data-dfn-for="MediaDevices" class="methods">
+        <dt>
+          <dfn>getDisplayMediaSet</dfn>
+        </dt>
+        <dd>
+          <p>
+            Prompts the user for permission to live-capture multiple of their
+            <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>.
+          </p>
+          <p>
+            The <a href="#dfn-user-agent">user agent</a> MUST let the end-user choose which
+            <a data-cite="screen-capture#dfn-display-surface">display surfaces</a> to share
+            out of all available choices every time, and MUST NOT use
+            any {{MediaTrackConstraints}} in <var>options</var>.<code>video</code>
+            or <var>options</var>.<code>audio</code> to limit that choice.
+          </p>
+          <p>
+            The <a href="#dfn-user-agent">user agent</a> MUST return a list of {{MediaStream}}
+            objects, where each stream belongs to exactly one of the
+            <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>
+            selected by the user. In case not all the
+            <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>
+            selected by the user can be captured, {{MediaDevices/getDisplayMediaSet}} must return
+            a promise <a>rejected</a> with a new {{InvalidStateError}}.
```

Comment:
Sure, I am open for suggestions. I am more in favor of implementing an "all or nothing" strategy. For a user it may be quite confusing if they shared 3 screens, but only 2 of them show up in the end (though for the specific failure case of an unplugged screen it may be more clear to them).

Reply:
> For a user it may be quite confusing if they shared 3 screens, but only 2 of them show up in the end

I'm not sure if the user would be more surprised/upset if the wrong number of screens shows up, or if nothing shows up.